tests/modules/users/resources/test_modifying_users_info.py

```diff
@@ -1,8 +1,17 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=missing-docstring
 import json
+from pathlib import Path
+import re
+import shutil
+import tempfile
 import uuid
 
+from app.modules.assets.models import Asset
```

Comment:
As before, I think the models imports were  function scope for a reason, not sure how much of a problem this is for tests but it's what I've been doing

Reply:
I don't think it's a problem for tests because something in `app` isn't going to import from `tests` so shouldn't cause any circular imports